develop/dev-guide-connection-parameters.md

```diff
@@ -34,6 +34,36 @@ It is a common practice that the connection pool size is well adjusted according
 
 The application needs to return the connection after finishing using it. It is recommended that the application uses the corresponding connection pool monitoring (such as **metricRegistry**) to locate connection pool issues in time.
 
+### Configure the lifetime of connections
+
+When a TiDB server shuts down or restarts for maintenance or due to a hardware failure, your client connections might be reset, which can lead to application disruptions. To avoid this, you should close and reconnect long-running database connections at least once a day.
```

Comment:
```suggestion
When a TiDB server shuts down or restarts(for example, during maintenance, scaling, or hardware failure), your existing client connections might be closed unexpectedly. If your application keeps connections open for too long, these disconnections can cause errors or short service interruptions.

To make your application more resilient, it’s best to **proactively recycle connections** in your connection pool instead of keeping them alive indefinitely. A common practice is to close and recreate each connection at least once every 24 hours (or more frequently for critical workloads).

Most connection pool libraries provide a parameter to control the maximum lifetime of a connection:
```